app/agents/voice/automatic/tools/shopify_buddy_test/get_order.py

```
@@ -131,8 +131,8 @@ async def get_shopify_orders(params: FunctionCallParams):
             
             await params.result_callback(transformed_orders)
         except httpx.HTTPStatusError as e:
-            logger.error(f"HTTP error calling Shopify API: {e.response.status_code} - {e.response.text}")
-            await params.result_callback({"error": f"Shopify API error: {e.response.status_code}", "details": e.response.text})
+            logger.error(f"Tool Error: [get_shopify_orders] HTTP error calling Shopify API: {e.response.status_code} - {e.response.text}")
+            await params.result_callback({"error": f"Tool Error: [get_shopify_orders] Shopify API error: {e.response.status_code}", "details": e.response.text})
         except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)